sonaranalyzer-dotnet/rspec/vbnet/Sonar_way_profile_no_hotspot.json

```diff
@@ -5,8 +5,12 @@
                      "S112",
                      "S114",
                      "S117",
+                     "S1048",
```

Comment:
This is generated by rspec.ps1, I guess this PR was the first to run it with the new changes we made yesterday

Reply:
As we have other changes which don't belong to this PR, I think it's better to have only the rule S3603 added here.